requirements.txt

```diff
@@ -2,4 +2,7 @@ git+https://github.com/huggingface/transformers # We need this for training, unt
 accelerate
 sentencepiece
 fastapi
-uvicorn
\ No newline at end of file
+uvicorn
+openapi-spec-validator
+bitsandbytes
+scipy
```

Comment:
do we need scipy?

Reply:
it is listed as a dependency for bitsandbytes for the llama models or something, idk. when I ran it without it bitsandbytes got mad.